services/app-api/src/emailer/emails/unlockContractCMSEmail.ts

```diff
@@ -51,21 +51,16 @@ export const unlockContractCMSEmail = async (
         packagePrograms
     )
 
-    const isContractAndRates =
-        contractRev.formData.submissionType === 'CONTRACT_AND_RATES'
-
     const data = {
         packageName,
         unlockedBy: updateInfo.updatedBy.email,
         unlockedOn: formatCalendarDate(updateInfo.updatedAt),
         unlockedReason: updateInfo.updatedReason,
         shouldIncludeRates:
             contractRev.formData.submissionType === 'CONTRACT_AND_RATES',
-        rateInfos:
```

Comment:
I found a bug where unlocking CONTRACT_ONLY contracts would error in the eta templates. Looks like there is an additional barrier in the resubmit templates that prevented it but it seemed simpler just to remove this check since the templates already check for if rateInfos.length > 0